tests/framework/extensions/kubeconfig/getpodlogs.go

```diff
@@ -0,0 +1,56 @@
+package kubeconfig
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	k8Scheme "k8s.io/client-go/kubernetes/scheme"
+	restclient "k8s.io/client-go/rest"
+)
+
+// Get pod logs is a helper function that resturns logs from a pod using rest client
```

Comment:
small nit pick `//GetPodLogs` also could we just name this file `podlogs.go`?

Reply:
done updated